src/ddprof_worker.cc

```diff
@@ -205,12 +211,19 @@ void *ddprof_worker_export_thread(void *arg) {
   DDProfWorkerContext *worker = (DDProfWorkerContext *)arg;
   // export the one we are not writting to
   int i = 1 - worker->i_current_pprof;
-
-  if (IsDDResFatal(
-          ddprof_exporter_export(worker->pprof[i]->_profile, worker->exp[i]))) {
+  uint32_t number_of_cycles = worker->persistent_worker_state->number_of_cycles;
```

Comment:
I was thinking the exporter can get killed / thread ended. I wanted the variable to reflect the current state during the export, yet still capture that an export happened if the thread is ended by the worker. 

I made it maybe slightly verbose for readability. It could have been:
```
uint32_t number_of_cycles = (worker->persistent_worker_state->number_of_cycles)++;
```

Do you think I should update this ?